Broaden 11f (#12)
CarterKoala/wca-regulations-wrc: wca-regulations.md

# <wca-title>WCA Regulations

> <version>Version: January 1, 2025

Reviewer: ```suggestion
<version>Version: January 1, 2024
```

    - 11e1) If a competitor is granted an extra attempt, the extra attempt must be scrambled using a different scramble sequence. This scramble sequence must be generated using a current official version of an official WCA scramble program (see [Regulation 4f](regulations:regulation:4f)).
    - 11e2) If a competitor is granted an extra attempt, the extra attempt should be done right after the attempt that caused it, and must replace the original regularly numbered attempt.
    - 11e3) If it is unclear whether an incident should result in an extra attempt, the competitor may be granted a provisional extra attempt that will be used only if it is later found that an extra attempt was appropriate (e.g. by a decision of the WRC).

> - 11f) Decisions about an incident may be supported with any available evidence, at the discretion of the WCA Delegate.

Author: ```suggestion
- 11f) Decisions about an incident may be supported with any available evidence, including video or photographic analysis, at the discretion of the WCA Delegate.
```

> - 11f) Decisions about an incident may be supported with any available evidence, including video or photographic analysis, at the discretion of the WCA Delegate.

Author: ```suggestion
- 11f) Decisions about an incident may be supported with video, photographic or audio evidence, at the discretion of the WCA Delegate.
```
A narrower version to address Carter's concern.
